08_leukemia_cell_lines/README.md

```diff
@@ -0,0 +1,11 @@
+# Experiment
+
+Leukemia patient sample
+
+# Files and folders
+
+Snakefile: used for running rock_roi_method (see https://github.com/imallona/rock_roi_method)
```

Comment:
thanks for the readme! does it mean the rockroimethod is replaced by a single snakefile? can you elaborate with the call used to run 07 (cell lines)?

Reply:
also, do yo run workflow.sh directly after? before? what about indexing? what should I run to reproduce your results?